packages/node-core/src/indexer/connectionPool.service.ts

```diff
@@ -185,19 +193,40 @@ export class ConnectionPoolService<T extends IApiConnectionSpecific<any, any, an
     return Object.keys(this.pool).length;
   }
 
-  async handleApiDisconnects(apiIndex: number, endpoint: string): Promise<void> {
-    logger.warn(`disconnected from ${endpoint}`);
-    delete this.pool[apiIndex];
+  async handleApiDisconnects(index: number) {
```

Comment:
I think it makes sense for this to be made private now. It only gets called internally and making it private will also ensure that we update the other SDKs properly